docs/data/whats-new/WhatsNewLayout.js

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+
+function BlogCard(props) {
+  return (
+    <Card
+      sx={{
+        margin: '10px',
+        flexBasis: '350px',
+        display: 'flex',
+        flexFlow: 'column',
+        boxShadow: '0px 1px 3px 0px rgba(0,0,0,0.25)',
+      }}
+    >
+      <CardContent sx={{ p: 0 }}>
+        <Box sx={{ borderBottom: '1px solid #eee', marginBottom: 1, px: 1 }}>
+          <Typography variant="caption" color="text.secondary">
+            {props.blog.announcementDate}
+          </Typography>
+          <Typography gutterBottom variant="h6" component="div">
+            {props.blog.title}
+          </Typography>
+        </Box>
+
+        <Box sx={{ p: 1 }}>
+          <Typography variant="body2" component="div" color="text.secondary">
+            {props.blog.description}
+          </Typography>
+
+          <List sx={{ listStyle: 'circle', pb: 0 }}>
+            {props.blog.highlightList.map((item) => (
+              <ListItem
+                sx={{
+                  py: 0.5,
+                  px: 0.5,
+                  [`&:before`]: {
+                    content: '"•"',
+                    color: 'grey',
+                  },
+                }}
+              >
+                <Typography sx={{ pl: 1 }} variant="body2">
+                  <Link sx={{ cursor: 'pointer' /*, color: '#818181'*/ }}>
+                    {item}
+                  </Link>
+                </Typography>
+              </ListItem>
+            ))}
+          </List>
+        </Box>
+      </CardContent>
+      <CardActions
+        sx={{ justifyContent: 'center', flex: '1 1 auto', alignItems: 'end' }}
+      >
+        <Button sx={{ width: '100%' }} href={props.blog.url}>
+          Read more
+        </Button>
+      </CardActions>
+    </Card>
+  );
+}
+
+export default function PickersPlaygroundWrapper() {
+  const blogs = [
```

Comment:
Just thinking out loud... might need to do some digging, but manually displaying the blog information/highlights like this can get messy real quick. Especially if plan to pump out blogs every so often. 

Im wondering if we should use a 3rd party service to host our blogs, which then automatically updates the "What's New Page" when published.

Reply:
I would make the connection with https://www.notion.so/mui-org/Improving-the-blog-infrastructure-bcae2b15493f4daba0758adf499f7793 where we talk about high level improvements for the blog.

Reply:
A headless CMS for the blog would be really nice! Then we could talk to the content API on this page and anywhere else we want to display updates. The folder that holds all of the company's blog posts is getting pretty unwieldy, and it's only going to get worse over time.